wfe/web-front-end_test.go

```diff
@@ -284,12 +394,11 @@ func TestIssueCertificate(t *testing.T) {
 		responseWriter.Body.String(),
 		"{\"type\":\"urn:acme:error:malformed\",\"detail\":\"Error unmarshaling certificate request\"}")
```

Comment:
Oh, i see, it's within the arguments of `test.AssertEquals`.  So it makes sense to be one more tab in.